draft-ietf-quic-v2.md

```diff
@@ -208,12 +208,35 @@ are willing to suffer a round-trip penalty if they are incorrect.
 # Applicability
 
 This version of QUIC provides no change from QUIC version 1 relating to the
-capabilities available to applications. Therefore, all Application Layer
+capabilities available to applications. Therefore, most Application Layer
 Protocol Negotiation (ALPN) ({{?RFC7301}}) codepoints specified to operate over
-QUICv1 can also operate over this version of QUIC.
+QUICv1 can also operate over this version of QUIC. Applications SHOULD NOT
+require new ALPNs for QUICv2 unless there is a specific performance or
+interoperability concern.
 
 All QUIC extensions defined to work with version 1 also work with version 2.
 
+## The HTTP/3 Exception
+
+The Alt-Svc HTTP field {{?RFC7838}} allows an HTTP server to inform clients of
+which versions of HTTP it supports. However, in order to use this information, clients
+also need to select a transport protocol that can carry HTTP.
```

Comment:
Slight nit. Talk of Alt-Svc this isn't outright wrong but it could be more accurate to say something like

HTTP Alternative Services (Alt-Svc) {{?RFC8738}} allow servers to nominate alternative means of interaction such as on a different host, port or protocol. Alt-Svc is the often used to advertise server support for HTTP/3 by including the "h3" ALPN identifier. When clients decide to connect to an HTTP/3 alternate service, they need to select a QUIC version to use.